docs/apb-filter-design.md

```diff
@@ -0,0 +1,80 @@
+# APB Filter Design Doc
+
+## High Level Summary
+
+Introduce 2 new fields to broker section of config:
+
+`whitelist: "/etc/ansible-service-broker/whitelist.yaml"`
+`blacklist: "/etc/ansible-service-broker/blacklist.yaml"`
+
+Both are optional file paths that are yaml files containing arrays
+of regexes. Broker will construct the set of eligible APBs from this list.
```

Comment:
This is part of the problem I mentioned to you. These aren't files anymore, they're arrays in the main configuration file on registries. What should I do now that implementation has changed since spec design?